CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ set(PROJECT_SOURCES
     searchrequest.cpp
     cardwindow.cpp
     cardscontainer.cpp
+    deckwindow.cpp
```

Comment:
klasy camel casem. Wiem, że tak qt tworzy klasy, ale to temu, bo tak masz go ustawionego